docs/.gitignore

```diff
@@ -18,12 +18,13 @@ report-check-links.xlsx
 /.vuepress/public/handsontable
 /.vuepress/public/@handsontable
 /.vuepress/public/data/
+!/.vuepress/dist
```

Comment:
The `/.vuepress/dist` directory should be ignored on develop or feature branches
```suggestion
/.vuepress/dist
```
Do you need to push dist files to the repo on the `prod-docs/latest` branch? Can't you just build the docs in the runner and upload it to Netlify without committing and pushing?

Reply:
We don't use Netlify deploy. Netlify works by downloading branch, not by deploing artifact. One goal of the task was to avoid build time on the netlify. Also currently i can use GUI to trigger netlify deployment without building

Reply:
Yes, because unfortunately dist is built but not committed. Netlify does checkout on the `prod-docs/latest` branch We don't want Netlify to use Netlify build time, we use it for the deployment.

Reply:
Fix was done the last week